conda/recipes/cudf/meta.yaml

```diff
@@ -51,6 +51,7 @@ requirements:
     - nvtx >=0.2.1
```

Comment:
Don't you need to increase the numba pin in this file as well?

Reply:
Fixed!